types/ziggy-js/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for ziggy-js 0.9
+// Project: https://github.com/tightenco/ziggy#readme
+// Definitions by: Ben Allfree <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/ziggy-js/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me